src/dkg_commitmentmatrix.erl

```diff
@@ -11,37 +12,43 @@
          deserialize/2
         ]).
 
--type row() :: {erlang_pbc:element(), erlang_pbc:element(), erlang_pbc:element()}.
--type matrix() :: {row(), row(), row()}.
--type serialized_row() :: {binary(), binary(), binary()}.
--type serialized_matrix() :: {serialized_row(), serialized_row(), serialized_row()}.
+-record(commitmentmatrix, {
```

Comment:
Some duplication here? Is it worth splitting this out into it's own dkg_matrix.erl with related helper functions?